releasenotes/notes/check_hostname_is_usable-dc4b99e6f7242b34.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Prevent a containerized agent from using the OS hostname as datadog agent hostname.
+    Datadog agent hostname should be stable and should reflect the identity of the host they are monitoring.
+    Container ID or kubernetes POD names are suitable for that purpose.
```

Comment:
```suggestion
    Container ID or kubernetes POD names are not suitable for that purpose.
```

Reply:
i guess that's what you meant? In the sense that we shouldn't use container id / pod name to represent the host.